node-bindings/dist/index.d.ts

```diff
@@ -0,0 +1,329 @@
+import { Block } from "./types";
```

Comment:
hmm this `node-bindings/dist` should have been ignored, I wonder if something went wrong with a current rebase?